src/SFA.DAS.LevyTransferMatching/Models/Pledge.cs

```diff
@@ -1,13 +1,16 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 
 namespace SFA.DAS.LevyTransferMatching.Models
 {
-    public class Pledge
+    public class Pledge : PledgeReference
```

Comment:
As per my previous comment about multiple uses of this class, but also I'm not convinced inheritence is really needed here, just to inherit a couple of properties.